lib/bundler/cli/install.rb

```diff
@@ -100,7 +100,9 @@ def run
         Bundler.ui.warn <<-WARN, :wrap => true
           Your Gemfile has no gem server sources. If you need gems that are \
           not already on your machine, add a line like this to your Gemfile:
-          source 'https://rubygems.org'
+          source 'https://rubygems.org' do
```

Comment:
Perhaps also reword above line from "add a line" to "wrap your Gemfile in a block like this" or something similar.